framework/src/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -957,6 +958,48 @@ public Void visitMethodInvocation(MethodInvocationTree node, Void p) {
     }
 
     /**
+     * A helper method to check that the array type of actual varargs is a subtype of the
+     * corresponding required varargs, and issues "argument.invalid" error for it's not a subtype of
+     * the required one.
+     *
+     * <p>Note this method requires that type checking for the each elements of varargs is executed
```

Comment:
Does that type checking need to happen before this one? Or is this only a statement about what this method guarantees.
Try clarifying this sentence.